timeless/__init__.py

```diff
@@ -46,6 +47,8 @@ def initialize_extensions(app):
     import timeless.items.models
     import timeless.employees.models
     import timeless.companies.models
+    #initialize celery
+    make_celery(app)
```

Comment:
@paulodamaso  Later to define celery task we will need `Celery` instance, so it should be like this `app.celery = make_celery(app)`. After we can use it the following way:
```python
from flask import current_app


@current.app.celery.shared_task()
def sync_something():
    pass
```